go1.x/cookiecutter-aws-sam-eventbridge-schema-app-golang/cookiecutter.json

```diff
@@ -0,0 +1,15 @@
+{
+    "project_name": "Your EventBridge Starter app",
+    "runtime": "go1.x",
+    "architectures": {
+        "value": []
+    },
+    "function_name": "HelloWorld",
+    "AWS_Schema_registry": "aws.events",
+    "AWS_Schema_name": "EC2InstanceStateChangeNotification",
+    "AWS_Schema_root": "aws.ec2.EC2InstanceStateChangeNotification",
+    "AWS_Schema_source": "aws.ec2",
+    "AWS_Schema_detail_type": "EC2 Instance State-change Notification",
+    "codegen_path": "{{ cookiecutter.project_name }}/{{ cookiecutter.function_name }}/{{ cookiecutter.AWS_Schema_root|lower|replace('.', '/') }}",
```

Comment:
The way that Golang does imports requires you to know the whole path from the GoRoot, while Python and Java have friendlier import semantics. This allows cookiecutter to pass in the right path to make things build right out of the box

Reply:
Interesting. Deeper question for why you need GoRoot? I thought `Modules` allowed you to not think about it at all but maybe there is something deeper here. I am not Go Expert, so more just curious.

Reply:
Sorry for the confusion here, this is not related to GoRoot. This is the path for the schema generated code from where the go module is specified (in this case {{ cookiecutter.project_name }}). To import the generated code into the main.go file you need to know the import path for the generated code. Since this is dynamic based on the schema that is downloaded, this is some post-processing to generate the path at runtime.

Reply:
Ahh. Ok. Thanks for explaining it, this makes more sense to me now.